OpenRA.Game/Traits/TraitsInterfaces.cs

```diff
@@ -266,6 +267,7 @@ public interface IRenderModifier
 		IEnumerable<Rectangle> ModifyScreenBounds(Actor self, WorldRenderer wr, IEnumerable<Rectangle> r);
 	}
 
+	public interface ICursorPalette : ITraitInfoInterface { string Palette { get; } ImmutablePalette ReadPalette(IReadOnlyFileSystem fileSystem); }
```

Comment:
This interface has more than one thing in it, so please use multiple lines for this definition.

Reply:
We probably want to call this `ICursorPaletteInfo` to signify that this is an *Info interface.